tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml

```diff
@@ -1,33 +1,38 @@
 trigger: none
 
 name: 'orttraining_ci_$(Date:yyyyMMdd)_$(Rev:r)'
-
-pool: 'Linux-CPU-CorpNet'
-
-variables:
-- group: pai-variables
-- name: pai-user
-  value: onnxrt
+pool: 'AMD-GPU'
```

Comment:
can i get rid of Linux-CPU-CorpNet now? :)

Reply:
O yes :)